config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+common: &common
+  secret_key_base: 'f0b774b4d06f7ea3357dec260cd6231367009d35525b3c437b27e299d52599bebd4eb1cd83fc3f69ff0d46a0510e5e4ba3bffd26b503b7713100f6083fc4baba'
+
+  smtp:
+    user_name: <%= ENV['SMTP_USER_NAME'] || 'owen' %>
```

Comment:
This really seems like bikeshedding. Could you make this change another PR after this is merged?